packages/snet_cli/snet_cli/commands/mpe_account.py

```diff
@@ -41,5 +40,10 @@ def transfer_in_mpe(self):
         self.transact_contract_command("MultiPartyEscrow", "transfer", [self.args.receiver, self.args.amount])
 
     def mint_token(self):
-        address = self.args.address
-        self.transact_contract_command("SingularityNetToken", "mint", [address, int(self.args.amount)])
+        rez = self.config.session_to_dict()
+        network = rez['session']['network']
+
+        if network == 'local':
+            self.transact_contract_command("SingularityNetToken", "mint", [self.args.address, int(self.args.amount)])
```

Comment:
can we add this in the test case itself and remove referece to mint_token def in mpe_account.py

Reply:
separated as test case